BasicSample/Assets/HandTracking/Scripts/OpenXRExtensionHandJointsManager.cs

```diff
@@ -0,0 +1,240 @@
+                        }
+                        handJointGameObjects[handJoint] = jointObject;
+                    }
+
+                    GameObject handJointGameObject = handJointGameObjects[handJoint];
+                    HandJointLocation handJointLocation = locations[(int)handJoint];
+                    handJointGameObject.transform.SetPositionAndRotation(handJointLocation.Pose.position, handJointLocation.Pose.rotation);
+                    handJointGameObject.transform.localScale = Vector3.one * handJointLocation.Radius;
+                }
+            }
+
+            /// <summary>
+            /// When this hand becomes inactive, it's best practice to hide the in-scene representation.
+            /// </summary>
+            public void DisableHandJoints()
+            {
+                if (handRoot != null)
+                {
+                    handRoot.SetActive(false);
+                }
+            }
+
+            /// <summary>
+            /// Destroys the hand representation when the hand needs to be cleaned up.
+            /// </summary>
+            public void DestroyHandJoints()
+            {
+                if (handRoot != null)
+                {
+                    Destroy(handRoot);
+                }
+            }
+
+            private static void ColorJointObject(GameObject jointObject, HandJoint finger, int? index)
+            {
+                Color jointColor = Color.magenta;
+                string fingerName = finger.ToString();
+
+                const string Thumb = "Thumb";
+                const string Index = "Index";
+                const string Middle = "Middle";
+                const string Ring = "Ring";
+                const string Little = "Little";
+
+                if (fingerName.StartsWith(Thumb))
+                {
+                    jointColor = (index == 0) ? Color.magenta : Color.red;
+                }
+                else if (fingerName.StartsWith(Index))
+                {
+                    jointColor = new Color(1.0f, 0.647f, 0.0f);
+                }
+                else if (fingerName.StartsWith(Middle))
+                {
+                    jointColor = Color.yellow;
+                }
+                else if (fingerName.StartsWith(Ring))
+                {
+                    jointColor = Color.green;
+                }
+                else if (fingerName.StartsWith(Little))
+                {
+                    jointColor = Color.blue;
+                }
+
+                // Lighten the colors as we go up the finger.
+                int i = index ?? 0;
+                float saturation = 1.0f - (i * 0.15f);
+                Color.RGBToHSV(jointColor, out float h, out float _, out float v);
+                jointObject.GetComponent<Renderer>().material.color = Color.HSVToRGB(h, saturation, v);
+            }
+
+            /// <summary>
+            /// These indices map to the order they're provided through Unity's HandData.
```

Comment:
this comment is not correct.  It is indices to map to the coloring of joints, right?

Reply:
Updated!